src/main/scala/org/neo4j/spark/util/Validations.scala

```diff
@@ -134,6 +141,43 @@ object Validations {
     }
   }
 
+  def checkOptionsConsistency(neo4jOptions: Neo4jOptions): Unit = {
```

Comment:
can you add just a few lines for describing what this method does internally? It took a while to me to understand the behaviour